src/gource.cpp

```diff
@@ -901,6 +901,7 @@ void Gource::reset() {
     mousedragged = false;
 
     commitqueue_max_size = 100;
+    starting_z = -300;
```

Comment:
You need to move this into the constructor before ZoomCamera() gets called. I think its getting a random value right now.